components/dashboards-web-component/src/viewer/DashboardViewPage.jsx

```diff
@@ -197,14 +197,35 @@ class DashboardViewPage extends Component {
         );
     }
 
+    /**
+     * Check if the given page is marked as hidden.
+     *
+     * @param {{*}} page Page object
+     * @returns {boolean} Is hidden
+     */
+    isPageHidden(page) {
+        return Object.prototype.hasOwnProperty.call(page, 'hidden') && page.hidden;
+    }
+
+    /**
+     * Render pages menu.
+     *
+     * @returns {XML} Html content
+     */
     renderPagesList() {
         const landingPage = this.dashboard.landingPage;
         const history = this.props.history;
         return this.dashboard.pages.map((page) => {
```

Comment:
With the new `hidden`, all `page` objects in the `pages` array won't produce `<ListItem>` element. Hence, instead of `Array.map()` lets use `Array.forEach()` here. Then you won't be needed to `return '';`.

Reply:
```js
const listItems = [];
return this.dashboard.pages.forEach((page) => {
     if (this.isPageHidden(page)) {
          return;
     }
     // create listItem = <ListItem>
     listItems.push(listItem);
}
```

Reply:
Ack. Thanks!